.github/workflows/kube-linter.yaml

```diff
@@ -2,19 +2,13 @@ name: KubeLinter
 
 on:
   push:
-    paths-ignore:
-      - '**.md'
```

Comment:
I think this was added for reason, if md and txt are not omitted - we might get unnecessary warnings

Reply:
Actually, I'm not correct, this config is not related to kube-linter settings.